packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -593,6 +593,12 @@ export interface DatabaseInstanceNewProps {
    * @default - None
    */
   readonly s3ExportBuckets?: s3.IBucket[];
+  /**
```

Comment:
Missing empty line before this line:

```suggestion

/**
```